docs/_templates/class_only_docstring.rst

```diff
@@ -0,0 +1,5 @@
+{{ fullname | escape | underline}}
```

Comment:
why do we need this? Ocean's other packages don't have it

Reply:
Suppose we'll need to add it to the sdk as well when we make the PR to update penaltymodel. I'll probably do a similar thing for dimod's typing.

Reply:
Realized that we only temporarily documented the types, so it can be removed.